src/constants/properties.constant.ts

```diff
@@ -153,4 +153,10 @@ export const PROPERTIES = {
       contractPercentage: 100,
     },
   },
+  FACILITY: {
+    DEFAULT: {
+      compBalPctReserve: 0,
+      compBalPctAmount: 0,
```

Comment:
We need to be careful here, the default for `compBalPctReserve` for the GET response (`0`) is not the same as for the POST request (`75`)

Reply:
Good spot, I've gone with nesting (i.e. `PROPERTIES.DEFAULT.FACILITY.GET.compBalPctAmount`).